sdk/src/types/api/core/response.rs

```diff
@@ -72,11 +85,88 @@ pub struct StatusResponse {
     serde(rename_all = "camelCase")
 )]
 pub struct MetricsResponse {
+    #[serde(with = "crate::utils::serde::string")]
     pub blocks_per_second: f64,
+    #[serde(with = "crate::utils::serde::string")]
     pub confirmed_blocks_per_second: f64,
+    #[serde(with = "crate::utils::serde::string")]
     pub confirmed_rate: f64,
```

Comment:
Ah few things got updated now https://github.com/iotaledger/tips-draft/pull/24/files/43b25b17b564bc42963d9f32645a98f1bc2a58c3..30fda3e548cca0704d13dcd4564dd0e74206a5a0
for example this one
```suggestion
    pub confirmation_rate: f64,
```